ocs_ci/ocs/ui/base_ui.py

```diff
@@ -836,8 +837,27 @@ def take_screenshot(driver):
     time.sleep(0.5)
 
 
-@retry(TimeoutException, tries=3, delay=3, backoff=2)
-@retry(WebDriverException, tries=3, delay=3, backoff=2)
+def garbage_collector_webdriver():
+    """
+    Garbage Collector for webdriver objs
+
+    """
+    collected_objs = gc.get_objects()
+    for obj in collected_objs:
+        if str(type(obj)) == constants.WEB_DRIVER_CHROME_OBJ_TYPE:
+            try:
+                obj.close()
+            except Exception as e:
```

Comment:
Discussed offline, this is too broad. Should be replaced with the explicit exceptions

Reply:
done